<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientProf</class>
 <widget class="QMainWindow" name="clientProf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(236, 229, 199);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="font">
        <font>
         <family>Lalezar</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(53, 66, 89);
</string>
       </property>
       <property name="text">
        <string>بازگشت</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="font">
        <font>
         <family>Lalezar</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(53, 66, 89);</string>
       </property>
       <property name="text">
        <string>سوابق خرید</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="font">
        <font>
         <family>Lalezar</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(53, 66, 89);</string>
       </property>
       <property name="text">
        <string>فروشگاه</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>751</width>
      <height>431</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Lalezar</family>
      <pointsize>20</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(53, 66, 89);</string>
    </property>
    <property name="title">
     <string> پروفایل</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>100</y>
       <width>91</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>نام کامل:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>170</y>
       <width>101</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>شماره تلفن:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>628</x>
       <y>230</y>
       <width>71</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>آدرس:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>100</y>
       <width>51</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>شهر:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>170</y>
       <width>91</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>نام کاربری:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>230</y>
       <width>71</width>
       <height>38</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string>پسورد:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);
background-color: rgb(116, 141, 166);</string>
     </property>
     <property name="text">
      <string>ذخیره تغییرات</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>10</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(53, 66, 89);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_5">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>160</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_13">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>91</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_14">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>231</y>
       <width>256</width>
       <height>181</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
     <property name="plainText">
      <string/>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;Lalezar&quot;;
background-color: rgb(116, 141, 166);</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>256</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lalezar</family>
       <pointsize>12</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(116, 141, 166);
font: 12pt &quot;Lalezar&quot;;</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
